src/python/dxpy/bindings/apollo/assay_filtering_json_schemas.py

```diff
@@ -0,0 +1,30 @@
+EXTRACT_ASSAY_EXPRESSION_JSON_SCHEMA = {
+    "annotation": {
+        "type": dict,
+        "properties": {
+            "feature_name": {"type": list, "required": False},
+            "feature_id": {"type": list, "required": False},
+        },
+        "conflicting_keys": [["feature_name", "feature_id"]],
+    },
+    "expression": {
+        "type": dict,
+        "properties": {"min_value": {"type": str}, "max_value": {"type": str}},
```

Comment:
We might want to consider introducing custom types here (like `numeric`). Also is it possible to receive an actual `int`? In which case a poly-type definition (like `"type": [str, int]`)

Reply:
We only need `str` for CLIEXPRESS. Otherwise, while it's possible to use any type with this `class` (beucase implementation is a Python dict), we will just error out with the more basic JSON checks/parsing if all keys and values are not `str` (wrapped in `"`).